andela/activo-api
# #159897990 An endpoint to delete a space

#### What does this PR do?
Implements an endpoint to delete a space

#### Description of Task to be completed?
As a developer, I should be able to delete a single space when I make a request to `/spaces/<space_id>`

**Given** I have a valid authentication token
**When** I make a delete request this endpoint `/spaces/<space_id>` - endpoint to a space
**Then** the space should be deleted
#### How should this be manually tested?

- create a local copy `git fetch origin ft-delete-space-159897990:ft-delete-space-159897990`
- checkout to the branch `git checkout ft-delete-space-159897990`
- Start the application with the command: `flask run`
on postman, send a `DELETE` request to `{{base-url}}/api/v1/spaces/<space_id>`
you should receive a response with status 200 and a message: `"<space_name> successfully deleted"`

#### What are the relevant pivotal tracker stories?
[159897990](https://www.pivotaltracker.com/story/show/159897990)

#### Screenshots 
<img width="1121" alt="screen shot 2018-08-30 at 17 48 01" src="https://user-images.githubusercontent.com/17172747/44859554-0cc61e00-ac7d-11e8-8fb9-0915941ae1c8.png">


#### Documentation
[Docs](https://www.getpostman.com/collections/fde586bbab0ab74bf9be)
